plugins/jobtype/src/azkaban/jobtype/HadoopConfigurationInjector.java

```diff
@@ -166,7 +166,7 @@ public static String getDirName(Props props) {
       }
     }
 
-    return "_resources_" + dirSuffix;
+    return "_resources_" + dirSuffix.replace(':', '_');
```

Comment:
Can you replace `"azkaban.flow.nested.path"` in line 160 above with `CommonJobProperties.NESTED_FLOW_PATH`?